### DISCUSSION TITLE
[Help Wanted] 2FA failing on attempting to sync existing  free account to new self-hosted instance

### DISCUSSION THREAD

**Discussion by dm5n** [standardnotes/forum#2907] [General]
I've setup SN on a subdomain of new Oracle Cloud Instance and confirmed at _subdomain.example.com_ with msg:

"Welcome to the Standard Notes server infrastructure. Learn more at https://docs.standardnotes.com"

But when I try to enter the subdomain into the Accounts menu in the web app, 2FA fails to verify.

Changingthe Custom Server back to api.standardnotes.com results in another 2FA request which then passes.

Using Google Authenticator app.

I'd like to be able to sync my existing account to new selfhosted server.  Also, is it possible to continue to use the SN free account in addition to self-hosting?

**Comment by effieeee** [comment]
Hi @dm5n, I'm afraid that free sync accounts that were created on our official server cannot be accessed on a self hosted instance, since they technically don't exist on your self hosted server yet 😞

You'll want to export a [backup file](https://standardnotes.com/help/14/how-do-i-create-and-import-backups-of-my-standard-notes-data) from your existing free account on our server, then import this file into a newly created account on your self hosted server 🙏

**Comment by dm5n** [comment]
Thankyou, that's logical 👍🏾

Could you clarify – do I need to setup account on the standardnotes web app or only on my own server?

**Comment by myreli** [comment]
Hi @dm5n, were you able to solve it? 

You should sign in to the standard notes webapp to retrieve the backup file, and then create an account in your own server and import there (also trough the client). The [self-host docs](https://standardnotes.com/help/self-hosting/getting-started) were recently updated, so that should help as well.
